packages/11ty/_plugins/figures/annotation-factory.js

```diff
@@ -0,0 +1,58 @@
+const chalkFactory = require('~lib/chalk')
+const path = require('path')
+const titleCase = require('~plugins/filters/titleCase')
+const { error } = chalkFactory('Figure Processing:IIIF:Annotations')
+
+/**
+ * Quire Annotation
+ */
+module.exports = class AnnotationFactory {
+  constructor(iiifConfig) {
+    this.iiifConfig = iiifConfig
+  }
+
+  get filepath() {
+    return this.figure.preset === "zoom"
+      ? [
+          this.iiifConfig.outputDir,
+          this.data.id,
+          this.iiifConfig.imageServiceDirectory,
+        ].join('/')
+      : [this.iiifConfig.inputDir, this.data.src].join('/');
+  }
```

Comment:
A JSDoc here to state that `preset` zoom serves the output of the iiif tiling processing output otherwise use the assets input directory.